XCM Refactor (#4932)
w3f/polkadot-wiki: docs/learn/learn-xcm-instructions.md

slug: ../learn-xcm-instructions
---

import Tabs from "@theme/Tabs";

This page can also be viewed at the [`xcm-format`](https://github.com/paritytech/xcm-format)
repository, where each instruction and register is explained in-depth.

:::caution

For a more practical approach to utilizing XCM registers and instructions, refer to the

> [XCM Docs](https://paritytech.github.io/xcm-docs/). Please keep in mind that these are currently
> under development.

Reviewer: ```suggestion
[XCM Docs](https://paritytech.github.io/xcm-docs/). Please keep in mind that XCM is
under active development.
```

- `UnlockAsset`
- `NoteUnlockable`
- `RequestUnlock`

## Instructions Application Example

The following presents the practical mapping of instructions to some of the core functionality to

> XCM [[sdf](./learn-xcm.md/#core-functionality-of-xcm)](./learn-xcm.md/#core-functionality-of-xcm).

Reviewer: ```suggestion
XCM [sdf](./learn-xcm.md/#core-functionality-of-xcm).
```
